backend/src/main/java/com/ody/route/service/RouteClient.java

```diff
@@ -0,0 +1,68 @@
+package com.ody.route.service;
+
+import com.ody.meeting.domain.Location;
+import com.ody.route.config.RouteProperties;
+import com.ody.route.domain.Duration;
+import com.ody.route.dto.CalculateDurationResponse;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import org.springframework.web.client.RestClient;
+
+public class RouteClient {
+
+    private final RouteProperties routeProperties;
+    private final RestClient restClient;
+
+    public RouteClient(
+            RouteProperties routeProperties,
+            RestClient.Builder routeRestClientBuilder
+    ) {
+        this.routeProperties = routeProperties;
+        this.restClient = routeRestClientBuilder.build();
+    }
+
+    public Duration calculateDuration(Location origin, Location target) {
+        CalculateDurationResponse response = restClient.get()
+                .uri(makeURI(origin, target))
+                .retrieve()
+                .body(CalculateDurationResponse.class);
+        return responseToDuration(response);
+    }
+
+    private URI makeURI(Location origin, Location target) {
+        String originLongitude = "?SX=";
+        String originLatitude = "&SY=";
+        String targetLongitude = "&EX=";
+        String targetLatitude = "&EY=";
+        String apiKey = "&apiKey=";
+
+        try {
+            return new URI(routeProperties.getUrl()
+                    + originLongitude + origin.getLongitude()
+                    + originLatitude + origin.getLatitude()
+                    + targetLongitude + target.getLongitude()
+                    + targetLatitude + target.getLatitude()
+                    + apiKey + routeProperties.getApiKey()
+            );
+        } catch (URISyntaxException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
[제안] StringBuilder를 사용하는 것은 어떤가요? 또한, `?SX=`, `&SY=` 등은 네이밍으로 의미를 전달하기 어렵고 재사용될 가능성도 낮으므로 변수로 선언하지 않아도 괜찮을 것 같아요

```suggestion
        StringBuilder uri = new StringBuilder();
        uri.append(routeProperties.getUrl())
                .append("?SX=").append(origin.getLongitude())
                .append("&SY=").append(origin.getLatitude())
                .append("&EX=").append(target.getLongitude())
                .append("&EY=").append(target.getLatitude())
                .append("&apiKey=").append(routeProperties.getApiKey());
        try {
            return new URI(uri.toString());
        } catch (URISyntaxException e) {
            throw new RuntimeException(e);
        }
```